setup.py

```diff
@@ -87,27 +90,16 @@ def build_extension(self, ext: Extension):
     packages=find_packages(exclude=["examples*", "tests*"]),
     classifiers=[
         "Programming Language :: Python :: 3 :: Only",
-        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: Implementation :: CPython",
         "Topic :: Software Development :: Libraries :: Python Modules",
     ],
     python_requires=">=3.6",
     extras_require={
-        "dev": [
-            "pytest>=6",
-            "pytest-mock>=3.5",
-            "mypy>=0.910",
-            "flake8>=3.8",
-            "black>=20.8b1",
-            "isort>=5.6",
-            "autoflake>=1.4",
-            "msgpack>=1.0.2",
-            "pre-commit>=2.15.0",
-            "httpx>=0.19.0",
-        ],
+        "dev": dev_requirements,
```

Comment:
Necessary to move it out and use a separate file? Since here we also have other dependencies, I feel it's better to put them at one place for easier viewing and reference.

After all they are just doing the same thing.

Reply:
Otherwise cannot only install the dependencies for the lint.